utils/CreateDataset.py

```diff
@@ -112,6 +141,19 @@ def __getitem__(self, index):
             metadata.update(sample_metadata)
             # where bandwise array has no data values, set as np.nan
             # sat_img[sat_img == metadata['nodata']] = np.nan # TODO: problem with lack of dynamic range. See: https://rasterio.readthedocs.io/en/latest/topics/masks.html
+
+        ######################  SINGLE-CLASS MODELS TEST  #######################
+
+        # Loop through classes that should be hidden (i.e. converted to background)
```

Comment:
Since you're using hdf5 files containing multiples classes, you bypass the "min_annotated_percent" parameter? You'd probably be passing a lot of samples with 0 values only, right?

Reply:
1. lines 144-156 shouldn't be part of this PR :)
2. Hmm let me give this a thought. I thought Vic's random sampler (oversampling samples containing buildings) corrected the imbalance afterwards, but you're probably right... :(